### PR TITLE
Export withHandlers() from frint-props

### PR DESCRIPTION
**Pull request by viacheslaff**
Addresses issue #33

### CODE REVIEW

File: packages/frint-props/src/index.spec.js [hunk 1]
```diff
@@ -5,4 +5,40 @@ describe('frint-props :: index', function () {
   test('is an object', function () {
     expect(typeof lib).toBe('object');
   });
+
+  test('compose is a function', function () {
```
[the remark this answers is not in the document]
Comment (fahad19) [reply packages/frint-props/src/index.spec.js]: 👍

[the remark this answers is not in the document]
Comment (viacheslaff) [reply packages/frint-props/src/index.spec.js]: @jackTheRipper, yeah, I'll update it

**Comment by viacheslaff** [comment]
Please review updated tests

File: packages/frint-props/src/index.spec.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jackTheRipper) [reply packages/frint-props/src/index.spec.js]: thx :)

**Pull request merged**